### DISCUSSION TITLE
Make notes protected by default + protected session duration

### DISCUSSION THREAD

**Discussion by anantone** [TriliumNext/Trilium#2952] [Q&A]
Hello,

Having recently started to use Trilium, I think it's really great, but there are two things that could make me adopt it fully:

I'm wondering if it is possible to make notes protected by default?

Also, can one edit the duration of a protected session (by default 10 minutes)?

I've been searching the wiki but couldn't find answers to these specific points. I would really appreciate some help figuring out if these settings can be adjusted.

PS: I'm using Version 0.52.3 thhrough the Cloudron package com.github.trilium.cloudronapp@1.6.1

**Comment by zadam** [comment]
Hi, 

> I'm wondering if it is possible to make notes protected by default?

Kind of. The protected status is "inherited" from the parent note when creating a new note. So if you make all notes protected (right click on root note -> Protect subtree), then any new note created will be protected by default (since any possible parent is also protected). This applies while in a protected session.

> Also, can one edit the duration of a protected session (by default 10 minutes)?

Yes, in Options -> Other -> Protected session timeout

**Reply by anantone** [reply]
Thank you very much for pointing that out, am only beginning to get situated with all the amazing features! I truly appreciate the spirit guiding this project!
